bootstrap.py

```diff
@@ -237,8 +239,18 @@ def get_json(url):
         request.add_header("Authorization", "Basic %s" % base64string)
         result = urllib2.urlopen(request)
         return json.load(result)
+    except urllib2.HTTPError, e:
+        if e.code == 401:
```

Comment:
I think this will eat all HTTPErrors that are not 401/422 and never display them. That sounds wrong.
Also it should say "Foreman", not "satellite" ;)